lib/rsc/story.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import React, { forwardRef } from "react";
+import { useStoryblokState } from "../common/client";
+import { ISbStoryData, StoryblokBridgeConfigV2 } from "../types";
+import { StoryblokComponent } from "./common";
+
+interface StoryblokStoryRSCProps {
+  story: ISbStoryData;
+  bridgeOptions: StoryblokBridgeConfigV2;
+  [key: string]: unknown;
```

Comment:
By adding `[key: string]: unknown` we are relaxing the type too much, it is accepting any property no matter the value type, I suggest we narrow it down

Reply:
this is to follow the style of the standard `StoryblokComponent` declaration, because those are the `restProps` that are passed down to the `<Component>` or the fallback component.

I agree they are a bit relaxed, do you think we should refactor this in this PR?

Reply:
It actually makes sense. At the end this component needs to accept any property and pass it down to child component, as it acting as a proxy component

Reply:
Yes, but I think we could list all the properties that we can accept at the lowest level. Probably something for a future refactor though?

Reply:
Totally, let's discuss separately 👍